### ISSUE TITLE
Bug: Windows Updates Sensor Does not Generate Unique IDs

### ISSUE THREAD

**Issue by zanix** [hass-agent/HASS.Agent#89]
**Describe the bug**

Using the Windows Updates sensor via the satellite service does not generate unique IDs causing only one computer to have this sensor in Home Assistant.

**To Reproduce**

1. Install and configure Hass.Agent for multiple computers
2. Configure Windows Update sensors in the satellite service using `windows_updates` for the Name and `Windows Updates` for the Friendly Name
3. Editing the server `sensors.json` may be required as the name and friendly name do not seem to save properly
    - #88 seems to address this

**Expected behavior**

Windows update sensors should have unique IDs and all computers' sensors should be available in Home Assistant.

**Screenshots**

```plain
2024-05-06 23:33:25.777 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID windows_updates_driver_updates_pending is already used by sensor.erebus_windows_updates_driver_updates_pending - ignoring sensor.nasya_windows_updates_driver_updates_pending
2024-05-06 23:33:25.778 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID windows_updates_software_updates_pending is already used by sensor.erebus_windows_updates_software_updates_pending - ignoring sensor.nasya_windows_updates_software_updates_pending
2024-05-06 23:33:25.780 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID windows_updates_driver_updates is already used by sensor.erebus_windows_updates_driver_updates - ignoring sensor.nasya_windows_updates_driver_updates
2024-05-06 23:33:25.781 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID windows_updates_software_updates is already used by sensor.erebus_windows_updates_software_updates - ignoring sensor.nasya_windows_updates_software_updates
2024-05-06 23:33:25.946 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID windows_updates_driver_updates_pending is already used by sensor.erebus_windows_updates_driver_updates_pending - ignoring sensor.ardena_windows_updates_driver_updates_pending
2024-05-06 23:33:25.949 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID windows_updates_software_updates_pending is already used by sensor.erebus_windows_updates_software_updates_pending - ignoring sensor.ardena_windows_updates_software_updates_pending
2024-05-06 23:33:25.951 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID windows_updates_driver_updates is already used by sensor.erebus_windows_updates_driver_updates - ignoring sensor.ardena_windows_updates_driver_updates
2024-05-06 23:33:25.952 ERROR (MainThread) [homeassistant.components.sensor] Platform mqtt does not generate unique IDs. ID windows_updates_software_updates is already used by sensor.erebus_windows_updates_software_updates - ignoring sensor.ardena_windows_updates_software_updates
```

**Misc info (please complete the following information):**
 - Windows build (ideally screenshot/info of `winver.exe` output):

![image](https://github.com/hass-agent/HASS.Agent/assets/1580378/226f192a-7055-4f57-8395-e17762a51a78)

 - Windows' UI language: English
 - HASS.Agent version: 2.1.0-beta1

Please check what's applicable (multiple answers possible):

 - [x] Installed via installer
 - [ ] Installed manually
 - [x] Problem occurs in HASS.Agent
 - [x] Problem occurs in Satellite Service

**Additional context**
None

**Logs**
If possible, please provide the logs of HASS.Agent and - if applicable - the Satellite Service. You can find HASS.Agent's logs by going to the main window, then `Configuration` -> `Logging` -> `open logs folder`. The Satellite Service's logs are in the same window, but the `Satellite Service` tab -> `open service logs folder`.

**Comment by amadeo-alex** [comment]
Replicated on the 2.1.0-beta2 branch, thanks for reporting and all the information.

**Comment by amadeo-alex** [comment]
Fixed in 2.1.0-beta2 :)
